packages/perseus/src/widgets/passage/passage-markdown.jsx

```diff
@@ -418,17 +419,20 @@ const parse: (string, ?ParseState) => Array<SingleASTNode> = (
 ) => {
     state = state || {};
     const paragraphedSource = source + "\n\n";
-    return builtParser(paragraphedSource, _.extend(state, INITIAL_PARSE_STATE));
+    return builtParser(
+        paragraphedSource,
+        _.extend(state, getInitialParseState()),
```

Comment:
I realize this is merged already, but this can be replaced easily with a spread (for future reference): 

```js
        {...state, ...getInitialParseState()}
```